pkg/compose/build.go

```diff
@@ -68,6 +69,27 @@ func (s *composeService) build(ctx context.Context, project *types.Project, opti
 		return nil, err
 	}
 
+	// Initialize buildkit nodes
+	var (
+		b     *builder.Builder
+		nodes []builder.Node
+	)
+	if buildkitEnabled {
+		builderName := options.Builder
+		if builderName == "" {
+			builderName = os.Getenv("BUILDX_BUILDER")
+		}
```

Comment:
Hmm, so we already do a look up in `cmd/build.go`:
https://github.com/docker/compose/blob/8318f66330358988a058bf611a953f464ab7973f/cmd/compose/build.go#L58-L61

But that won't apply for `up --build` (while this will).

@glours Do you think that `--builder` should become a global flag on Compose, and let the CLI/cmd part handle the env var lookup?

[I don't think we need to resolve this as part of this PR]